integrations/SD_Lon/sd_changed_at.py

```diff
@@ -96,11 +90,6 @@ def __init__(self, from_date, to_date=None):
         facet_info = self.helper.read_classes_in_facet('association_type')
         self.association_uuid = facet_info[0][0]['uuid']
 
-        # Create non-existent departments
-        logger.info('Check for new departments')
-        self.check_non_existent_departments()
-        self.fix_departments()
-
     def _add_profession_to_lora(self, profession):
```

Comment:
Når vi går direkte på lora er det så fordi vi ikke kan gøre det igennem Mo, eller er det fordi det er lettere?

Reply:
Mo indeholder intet api for at tilføje klasser, kun til at læse dem.